src/drachtio.cpp

```diff
@@ -208,6 +208,10 @@ namespace drachtio {
           sip->sip_cseq->cs_method_name) ;
         str.append("|") ;
         str.append( boost::lexical_cast<std::string>(sip->sip_cseq->cs_seq) ) ;
+        if (sip->sip_via && sip->sip_via->v_branch) {
```

Comment:
I'm wondering, is there an expected case where this is false?  I suppose we could get a request from a very old client that didn't add a branch (?)